getting-started/templates/sizing-examples/systems-sizing-example.yaml

```diff
@@ -358,9 +358,12 @@ tags:
     limits:
       memory: 8Gi
 
-  ## @param tags.replicaCount Number of pod replicas to deploy.
-  ## This service does not have auto-scaling. Set this value according to the needs of your environment.
-  replicaCount: 1
+  ## @param replicaCount Number of pod replicas to deploy. It is recommended to use an odd number.
+  ## This service does not have auto-scaling. Workload and tag data will be distributed across the replicas based on the number of workspaces. 
+  ## <ATTENTION> - Environments with high workspace cardinality and more than 240k Tags may require more replicas to balance the load. 
+  ## Set this value according to the needs of your environment.
+  ##
+  replicaCount: 3
```

Comment:
Let's comment this out, so by default customers will go with helm defined default. Take a look at execution resource profiles, we can say "uncomment to adjust .."